node_modules/oae-preview-processor/tests/test-previews.js

```diff
@@ -525,6 +525,13 @@ describe('Preview processor', function() {
                                 assert.ok(_.find(previews.files, function(file) { return file.filename === 'page.1.html'; }));
                                 assert.ok(!_.find(previews.files, function(file) { return file.filename === 'page.2.html'; }));
 
+                                // The PDF has 1 page, there should only be 1 corresponding txt file
```

Comment:
Shouldn't we have a test for a multi-page PDF where we verify that multiple `page.x.txt` files?